src/app/pages/subjects/subjects.scss

```diff
@@ -0,0 +1,155 @@
+@import 'variables';
+
+$light-gray: #f5f5f5;
+$lightest-gray: #fafafa;
+$light-line: #e0e0e0;
+$text-black: rgba(0, 0, 0, 0.87);
+$text-black2: rgba(0, 0, 0, 0.7);
+$text-black3: rgba(0, 0, 0, 0.54);
+
+.text-content {
+    color: $text-black;
+    font-family: Lato;
+
+    h1 {
+        font-weight: 900;
+    }
+
+    header {
+        background: $page-top-gradient;
+        font-size: large;
+        padding: 5rem 10rem;
+        text-align: center;
+    }
+
+    .subhead {
+        font-size: 1.5rem;
+    }
+
+    .feature-block {
+        display: flex;
+        font-size: initial;
```

Comment:
Yes, if you toggle it in the debugger, the font gets smaller. Off the top of my head, I don't know where it's being changed, either.